source/blog/2016-05-12-gaad.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Global Accessibility Awareness Day
+author: Nathan Hammond
+tags: Recent Posts, Ember A11y
+---
+
+May 19, 2016 marks the fifth annual **[Global Accessibility Awareness Day (GAAD)](http://www.globalaccessibilityawarenessday.org/)** and this year we're encouraging you, the Ember.js community, to participate. Not breaking the web has always been a requirement for building ambitious web applications and to that end we must ensure that our framework, addons, and the applications we collectively build maintain are accessible.
```

Comment:
I feel like absolute silence after clicking on a link counts as "breaking the web." But I do agree that I could tighten this up. Rewriting a bit.